code-studio/build-js.js

```diff
@@ -7,33 +7,31 @@ var build_commands = require('../shared/js/build-commands');
 var commander = require('commander');
 
 /** @const {string} */
-var SRC_PATH = './src/js/';
-
-/** @const {string} */
 var BUILD_PATH = './build/js/';
 
-/**
- * Files to build, given as paths rooted at code-studio/src/js/
- * Each will result in an output file.
- * @type {string[]}
- */
-var FILES = [
```

Comment:
I do, though I'm not sure I can put into words why. Seems like instead of commenting every usage, we should just be sufficiently commenting the common place (in build-commands).